packages/braid-design-system/src/lib/components/MenuRenderer/MenuRenderer.screenshots.tsx

```diff
@@ -97,28 +100,24 @@ export const screenshots: ComponentScreenshot = {
     {
       label: 'Placement bottom',
       Example: () => (
-        <Box display="flex">
-          <Box position="relative">
-            <Placeholder height={triggerHeight} label="Menu trigger" />
-            <Menu {...defaultProps} placement="bottom">
-              <MenuItem onClick={() => {}}>Item</MenuItem>
-              <MenuItem onClick={() => {}}>Item</MenuItem>
-            </Menu>
-          </Box>
+        <Box position="relative">
+          <Placeholder height={triggerHeight} label="Menu trigger" />
+          <Menu {...defaultProps} placement="bottom">
+            <MenuItem onClick={() => {}}>Item</MenuItem>
+            <MenuItem onClick={() => {}}>Item</MenuItem>
+          </Menu>
         </Box>
```

Comment:
These screenshots have changed, was there a reason to remove the flex wrapper here?